com.unity.ml-agents/Documentation~/filter.yml

```diff
@@ -0,0 +1,17 @@
+apiRules:
+- exclude:
+    uidRegex: ^MLAgents\.Tests$
+    type: Namespace
```

Comment:
Yes, but I'm holding off on that to avoid creating confusion on the master branch (at least for as long as possible). Go ahead with this and I'll update (or redo) the other one later.